README.md

```diff
@@ -469,11 +469,12 @@ models and reinforcement learning. Finally,
 [Objax](https://github.com/google/objax) is a minimalist object-oriented
 framework with a PyTorch-like interface.
 
-DeepMind has open-sourced an ecosystem of libraries around JAX including
-[Haiku](https://github.com/deepmind/dm-haiku) for neural network modules,
-[Optax](https://github.com/deepmind/optax) for gradient processing and
-optimization, [RLax](https://github.com/deepmind/rlax) for RL algorithms, and
-[chex](https://github.com/deepmind/chex) for reliable code and testing.
+DeepMind has open-sourced an
+[ecosystem of libraries around JAX](https://deepmind.com/blog/article/using-jax-to-accelerate-our-research) including [Haiku](https://github.com/deepmind/dm-haiku) for neural
+network modules, [Optax](https://github.com/deepmind/optax) for gradient
+processing and optimization, [RLax](https://github.com/deepmind/rlax) for RL
+algorithms, and [chex](https://github.com/deepmind/chex) for reliable code and
+testing.
```

Comment:
Small suggestion to aid those users who don't like reading (😄) - add a YouTube link to DeepMind's talk at NeurIPS.

@tomhennigan WDYT?

```suggestion
testing. (Watch the NeurIPS 2020 JAX Ecosystem at DeepMind talk [here](https://www.youtube.com/watch?v=iDxJxIyzSiM))
```

Reply:
Thanks for the suggestion, sgtm.